tests/test_prediction.py

```diff
@@ -235,6 +258,42 @@ def test_non_existent_taxonomy_category_gt_upload(model_run):
     )
 
 
+def test_scene_category_pred_upload(model_run):
+    prediction = SceneCategoryPrediction.from_json(
+        TEST_SCENE_CATEGORY_PREDICTIONS[0]
+    )
+    response = model_run.predict(annotations=[prediction])
+
+    assert response["model_run_id"] == model_run.model_run_id
+    assert response["predictions_processed"] == 1
+    assert response["predictions_ignored"] == 0
+
+
+def test_default_scene_category_pred_upload(model_run):
+    prediction = SceneCategoryPrediction.from_json(
+        TEST_DEFAULT_SCENE_CATEGORY_PREDICTIONS[0]
+    )
+    response = model_run.predict(annotations=[prediction])
+
+    assert response["model_run_id"] == model_run.model_run_id
+    assert response["predictions_processed"] == 1
+    assert response["predictions_ignored"] == 0
+
+
+def test_non_existent_taxonomy_scene_category_pred_upload(model_run):
+    prediction = SceneCategoryPrediction.from_json(
+        TEST_NONEXISTENT_TAXONOMY_SCENE_CATEGORY_PREDICTIONS[0]
+    )
+    response = model_run.predict(annotations=[prediction])
+    assert response["model_run_id"] == model_run.model_run_id
+    assert response["predictions_processed"] == 0
+    assert response["predictions_ignored"] == 0
+    assert (
+        f'Taxonomy {TEST_NONEXISTENT_TAXONOMY_SCENE_CATEGORY_PREDICTIONS[0]["taxonomy_name"]} does not exist in dataset'
+        in response["errors"][0]
+    )
```

Comment:
Again too specific.
```suggestion
```
You could maybe assert that `len(response["errors"]) > 0`?